pages/manage/productions/index.vue

```diff
@@ -141,30 +143,45 @@
       },
       async handleAddEventToProduction(eventId) {
         this.isAddingEventToProduction = true;
-        this.hasAddingEventToProductionError = false;
-        const res = await this.$api.productions.addEventById(
+
+        const foundEvent = this.events.find(
+          (event) => parseEventId(event['@id']) === eventId,
+        );
```

Comment:
This approach feels a bit convoluted to me. If I'm not mistaken you already get an error message back from the API, but it's just no translated yet so you try to guess the error by seeing if the event is already part of the production. But there could be other errors. For example the event can not exist, the event can already be part of another production, ... And in that case you do show the untranslated error message anyway. So I would simplify this to just show the untranslated error message, and deal with the localization of error messages another time. It's a more widespread problem than this specific place.

Reply:
we don't get an error message back from the API for this case, so maybe that is the fix needed rather than a frontend fix. 
When I add an event 'Hobbit 1' to the production 'The Hobbit' while it is already there, I get a 204 instead of a 400. Whereas for 'it already belongs to another production' it does work. And then the code without my addition should work I think

Reply:
Okay let's fix that on the backend then 👍 We still need some changes from this PR to actually show the messages from the backend right? Maybe remove this extra logic already and we'll fix it on the backend asap.

Reply:
I've added a ticket for the backend change https://jira.uitdatabank.be/browse/III-3481